dotcom-rendering/src/web/components/PulsingDot.importable.tsx

```diff
@@ -34,9 +35,10 @@ const animate = css`
 
 interface Props {
 	colour?: string;
+	diameter?: number;
```

Comment:
It feels like the size of the dot is always going to want to be coupled with the size of the text so maybe it makes more sense to have a `size` token here?

Reply:
Do you think adding a size prop with these values `small | medium | large` would be OK?

Reply:
It's very subjective so 🤷 . But I've listed some things I think play into this decision in my other comment above. Your call though!